### ISSUE TITLE
Align by submeshes

### ISSUE THREAD

**Issue by DrkWzrd** [CloudCompare/CloudCompare#1822]
**Describe the feature you would like**
The title is self descriptive. This is necessary if not exists.

![image](https://github.com/CloudCompare/CloudCompare/assets/40547839/e058f113-84ea-4b6e-9312-5a0e19877f0c)

**Is your feature request related to a problem? Please describe.**

The only way I found is "align" (does not align, only computes the matrix and throws error) the submeshes and then use the matrix in Edit>Apply Transform. But it's so tedious and it's not 100% exact (and must be 100% exact because both submeshes are identical).
If cloud compare can't do it, is there some software that can do it?

**Comment by dgirardeau** [comment]
Submeshes are a bit 'virtual' for now, in the sens that it's mostly a concept used for display only (since they are sharing the same set of vertices).

What you can do is to use the 'scissors' tool to isolate the submesh you are interested in, and then use the resulting mesh entity (the top level one, not the submesh).

**Comment by DrkWzrd** [comment]
The problem is if I cut the element, and I align it, it only aligns the cut part, I want to align all the mesh only using the data of the submesh. The important is I want all "the pole" to be aligned in axis and "vertical" position with the marker

![image](https://github.com/CloudCompare/CloudCompare/assets/40547839/389ccf0a-9098-443c-8d3f-e66aa05742bd)

**Comment by dgirardeau** [comment]
Indeed, you'll still have to copy the resulting matrix and apply it to the original mesh with 'Edit > Apply transformation matrix'...
